src/lib/fetch.ts

```diff
@@ -74,6 +76,9 @@ async function handleRes(res: Response) {
   // If we receive a non-JSON response, it's likely an error.
   // Let's debug the raw response body and throw it.
   const body = await res.text();
+  if (res.status === SUCCESS_NO_CONTENT) {
```

Comment:
Can you move this above the `res.text()` call? We don't need to call that if we're going to return immediately here with an empty object.

Reply:
Done